src/ui/server/controllers/insurance/eligibility/end-buyer/index.test.ts

```diff
@@ -136,10 +136,21 @@ describe('controllers/insurance/eligibility/end-buyer', () => {
         expect(req.session.submittedData).toEqual(expected);
       });
 
-      it(`should redirect to ${CHECK_YOUR_ANSWERS}`, () => {
+      it(`should redirect to ${PARTY_TO_CONSORTIUM}`, () => {
         post(req, res);
 
-        expect(res.redirect).toHaveBeenCalledWith(CHECK_YOUR_ANSWERS);
+        expect(res.redirect).toHaveBeenCalledWith(PARTY_TO_CONSORTIUM);
+      });
+
+      describe("when the url's last substring is `change`", () => {
+        it(`should redirect to ${CHECK_YOUR_ANSWERS}`, async () => {
+          req.originalUrl = END_BUYER_CHANGE;
+
+          await post(req, res);
+
+          const expected = CHECK_YOUR_ANSWERS;
+          expect(res.redirect).toHaveBeenCalledWith(expected);
+        });
```

Comment:
Can we add some E2E tests for changing the answers? `eligibility/change-your-answers/*.spec.js`

Reply:
e2e-tests\insurance\cypress\e2e\journeys\eligibility\change-your-answers\change-your-answers-end-buyer.spec.js already exists

Reply:
I mean for the new forms/fields we've added, `IS_PARTY_TO_CONSORTIUM`, `IS_MEMBER_OF_A_GROUP`

Reply:
Not in this PR, check your answers has not been made yet for this bit